### DISCUSSION TITLE
security and access problem with latest version

### DISCUSSION THREAD

**Discussion by jimsaville** [burnsoftnet/burnsoftnet.github.io#2] [General]
I was using the MGC initial v6.5.x.  I upgraded to the latest release (6.9.14.1) today.  Since then I have been unable to access the database.  The software will not accept my username and password.  I attempted to use the "forgot password" function and get a "Invalid length for a Base-64 char array or string.

As a test, I installed the most recent version on a different computer with a clean install.  Without making any other changes, I activated the security function and set a username, password and phrase.  I exited the software and re-entered.  The software would not accept my just created username and password.

How can I access my data?

**Comment by burnsoftnet** [comment]
Do you have ms access on your machine?  I’ll also put this issue down

**Reply by jimsaville** [reply]
I have Access M365.  I attempted to open the database directly and it wanted a password.

**Reply by burnsoftnet** [reply]
Use password. 14un0t2n0

Open up Owner_Info table. And set the usepwd column from 1 to 0

**Comment by jimsaville** [comment]
That worked.  Thank you.

I have another issue to report.  Should I do it in a separate discussion?

On Sat, Sep 3, 2022 at 7:50 AM BurnSoft ***@***.***> wrote:

> Use password. 14un0t2n0
>
> Open up Owner_Info table. And set the usepwd column from 1 to 0
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/burnsoftnet/burnsoftnet.github.io/discussions/2#discussioncomment-3542287>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A24CY7WAAQFQIGGXKYV2SH3V4M3RPANCNFSM6AAAAAAQDUD4MA>
> .
> You are receiving this because you authored the thread.Message ID:
> <burnsoftnet/burnsoftnet.github.io/repo-discussions/2/comments/3542287@
> github.com>
>


-- 
Jim Saville
Kaleidoscope Systems
301-346-5406
***@***.***

"The game of life is the game of boomerangs. Our thoughts, deeds and words
return to us sooner or later, with astounding accuracy."
- Florence Shinn -

**Reply by burnsoftnet** [reply]
Yes please and thank you for your findings. The upgrade in code was bound to leave some glitches that was. Over looked

**Reply by burnsoftnet** [reply]
New Release v6.9.15 Should resolve this issue.  https://github.com/burnsoftnet/MyGunCollection/releases/tag/v6.9.15.2
